shiptendance/app.js

```diff
@@ -0,0 +1,73 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+
+var routes = require('./routes/index');
```

Comment:
Remember, according to the prompt you want all these routes to be prefixed with `/pirates`, so you know what entity you're working with in the URL. Also, since you are not using users in this app, you can delete that line.